backend/conf/copr-backend.sysusers.conf

```diff
@@ -1,3 +1,3 @@
 #Type Name     ID             GECOS                 Home directory Shell
-u     copr     -              'COPR user'           -              /bin/bash
+u     copr     -              'COPR user'           /home/copr              /bin/bash
```

Comment:
I _think_ this deserves cleanup long-term, too.  `/home/copr` is sub-optimal for service accounts (we do the same right now for copr-dits-git).  FTR, also `copr-frontend` uses `/usr/share/copr/coprs_frontend`.

But I don't think we want to change this now in this PR - moving to `/usr/share/<something>` .  Opening https://github.com/fedora-copr/copr/issues/3812